.github/ISSUE_TEMPLATE/bug-issue-template.yml

```
@@ -48,7 +48,7 @@ body:
 - type: checkboxes
   attributes:
     label: Operating System
-    description: Which operating system have you seen the bug occur on
+    description: Which operating system have you seen the bug occurred on?
```

Please refactor to the suggestion below

```suggestion
    description: Which operating system have you seen the bug occur on?
```